### ISSUE TITLE
Async not working with CALL

### ISSUE THREAD

**Issue by xinyifly** [astahlman/ob-async#11]
```
#+NAME: sleep
#+BEGIN_SRC shell :async
sleep 2 && pwd
#+END_SRC

#+RESULTS: sleep
: eae1247455e5d35b9eb36671b10a317a

#+CALL: sleep()

#+RESULTS:
: /root
```

**Comment by astahlman** [comment]
Thanks for reporting this. @stnutt proposed a fix in #10 which would address this behavior.

**Comment by astahlman** [comment]
@stnutt fixed this in #10 

Confirmation:

```
#+NAME: sleep
#+BEGIN_SRC shell :async
sleep 2 && pwd
#+END_SRC

#+RESULTS: sleep
: /home/astahlman

#+CALL: sleep()

#+RESULTS:
: /home/astahlman
```

**Comment by GregorySchwartz** [comment]
Does not work when the call has a name, like:

```
#+NAME: test
#+CALL: sleep()
```

**Comment by astahlman** [comment]
Just confirming that I can reproduce.

**Comment by astahlman** [comment]
@GregorySchwartz  @xinyifly https://github.com/astahlman/ob-async/pull/48 will fix this, I think. I've added a test case for this scenario here: https://github.com/astahlman/ob-async/pull/48/files#diff-e2f9b3446fa07abcd51922f95ce941bcR256

**Comment by astahlman** [comment]
Marked as fixed by #48 - feel free to reopen if you still see this issue after pulling the latest version.
